src/pothole_detection/YOLOv4/PyTorch_YOLOv4-train/utils/general.py

```diff
@@ -18,12 +18,13 @@
 import yaml
 
 from utils.google_utils import gsutil_getsize
-from utils.metrics import fitness, fitness_p, fitness_r, fitness_ap50, fitness_ap, fitness_f   
+from utils.metrics import fitness, fitness_p, fitness_r, fitness_ap50, fitness_ap, fitness_f
```

Comment:
Just a general note, if you could rebase off of `origin/main`, it will do all this formatting stuff for you.

```sh
git fetch -p
git checkout vora/pothole  # or whatever branch you're one
git stash
git rebase -i origin/main
git push --force-with-lease
git stash pop   # use 'git stash list' to see if you have any other stashed changes

# wait a bit for the linter to work
git pull
```

When you push, it should automatically reformat everything with the Black Python formatter (as a Github Action). Conversely, you could pip install black and format your changes manually

Reply:
Oh it looks like you already did this! 👍

Reply:
Actually, you may have linted manually, because there should be a linter that runs on your MR if you're based on `origin/main` 🤔